package.json

```diff
@@ -84,6 +84,7 @@
     "node-bourbon": "^4.2.8",
     "node-notifier": "^4.5.0",
     "node-sass": "^3.7.0",
+    "react-collapsible": "^2.0.4",
```

Comment:
should be in dependencies rather than devDependencies

Reply:
Or both? Why did CI pass with it in devDependencies?